pages/index.js

```diff
@@ -36,50 +37,60 @@ export default function Index(props) {
     updateUsername()
   }
 
+  const checkVerification = async () => {
+    const user = await fetch(`api/user/${localStorage.userId}`).then(res => res.json());
+    if (user.isVerified === false) {
+      setIsVerified(false);
+    }
+  };
+
   useEffect(() => {
     // On mount, we check if a user is logged in.
     // If so, we'll retrieve the authenticated user's profile.
     magic.user.isLoggedIn().then((magicIsLoggedIn) => {
       if (magicIsLoggedIn) {
-        magic.user.getMetadata()
-          .then(setUserMetadata)
+        props.setUserEmail(localStorage?.email);
+        props.setUsername(localStorage?.name);
+        checkVerification();
       } else {
         // If no user is logged in, redirect to `/login`
         Router.push('/login');
       }
-    })
-    props.setUserEmail(userMetadata?.email || localStorage?.email)
-    props.setUsername(localStorage?.name)
+    });
   }, []);
 
-  return (userMetadata && (props.username || localStorage.name)) ? (
-    <div className='container' className={Styles.index}>
-      <Prompts
-        props={props}
-        userMetadata={userMetadata}
-      />
-    </div>
-  ) : (!props.username || !localStorage.name) ? (
-    <div className={Styles.index}>
-      <img src="https://i.imgur.com/NxwZ9x0.png" width="100px" height="100px" className={Styles.logo} />
-      <div className={Styles.welcome}>
-        <form className={Styles.welcomeForm}>
-          <h2>What's your name?</h2>
-          <input
-            placeholder="My name is..."
-            onChange={handleChange}
-            className={Styles.input}
-          />
-          <input
-            type="submit"
-            value="Get started"
-            onClick={handleSubmit}
-            className={Styles.button}
-          />
-        </form>
+  return (props.userEmail
+    ? (!props.username
```

Comment:
This is interesting syntax! I've never seen it before. Can you explain it to me, please? Looks cool? Confused and delights lol